components/config-provider/index.en-US.md

```diff
@@ -43,3 +43,4 @@ Some component use dynamic style to support wave effect. You can config `csp` pr
 | getPopupContainer | to set the container of the popup element. The default is to create a `div` element in `body`. | Function(triggerNode) | `() => document.body` | 3.11.0 |
 | locale | language package setting, you can find the packages in [antd/es/locale](http://unpkg.com/antd/es/locale/) | object | 3.21.0 |
 | prefixCls | set prefix class | string | ant | 3.12.0 |
+| direction | to set direction of all elements underneath itself, set `rtl` to view underneath components right-to-left styled | string: 'ltr', 'rtl', 'auto' | ltr | 3.21.5 |
```

Comment:
* Move Cascader demo into ConfigProvider.
* `set direction of text. See [demo](#your demo link)`

Reply:
We can add more component demo here.

Reply:
all done.